app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
   has_many :likes, foreign_key: 'author_id'
```

Comment:
- [ ] Hi, Reading from the requirements, it says under **install and setup devise** section: Ask for confirmation email.
From this, I will advise you to enabled `confirmable`.